content/kotlin/concepts/strings/endsWIth/endsWith.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.endsWith()'
+Description: 'Returns a boolean value representing wether the sequence ends with the specified suffix.' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: 
+  - 'String'
+  - 'Comparison'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.endsWith()`** method will compare the end of the sequence with the given suffix, returning a boolean value representing wether they are the same or not.
+
+## Syntax
+
+```pseudo
+String.endsWith(suffix)

```

Comment:
```suggestion
String.endsWith(suffix: String, ignoreCase: Boolean = false): Boolean
```
Mention the `ignoreCase` parameter. By default, it stays `false`, which means this function is case-sensitive. However, if we want it to be case-insensitive, we can set `ignoreCase` to `true`.